2-mandatory/3-financial-times.js

```diff
@@ -14,24 +20,48 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
-}
+    let lengthOfTitles = []
+    for(let i in allArticleTitles){
+        lengthOfTitles.push(allArticleTitles[i].split(" ").length)
+    }
+    let minimumTitlesIndex = lengthOfTitles.indexOf(Math.min(...lengthOfTitles))
```

Comment:
This is a little hard to read but you did it right. You created an array of words and then got the minimum and returned it.